content-services/7.1/support/index.md

```diff
@@ -106,7 +106,6 @@ Choose a combination of products to build your own Supported Stack. If anything
 | Alfresco Content Services SDK 4.3 | |
 | | |
 | **Applications** | |
-| Alfresco Digital Workspace 2.8 | |
 | Alfresco Digital Workspace 2.7 | |
```

Comment:
I think we should delete this line as well, since the ADW 2.7 Release Notes state a *minimum* version of ACS 7.2. This is also published in the [Alfresco Product Support Status page](https://www.alfresco.com/services/subscription/technical-support/product-support-status).
```suggestion
```